.github/workflows/build.yml

```diff
@@ -248,17 +256,18 @@ jobs:
             --mount "type=bind,src=$PWD/signed,dst=/doi-build/signed"
 
             # https://explore.ggcr.dev/?repo=docker/image-signer-verifier
-            docker/image-signer-verifier:0.3.3@sha256:a5351e6495596429bacea85fbf8f41a77ce7237c26c74fd7c3b94c3e6d409c82
+            "docker/image-signer-verifier:$BASHBREW_IMAGE_SIGNER"
 
             sign
 
-            --envelope-style oci-content-descriptor
-
-            --aws_region "$AWS_KMS_REGION"
-            --aws_arn "awskms:///$AWS_KMS_KEY_ARN"
+            --kms="AWS"
+            --kms-region="$AWS_KMS_REGION"
+            --kms-key-ref="$AWS_KMS_KEY_ARN"
 
-            --input oci:///doi-build/unsigned
-            --output oci:///doi-build/signed
+            --input=oci:///doi-build/unsigned
+            --output=oci:///doi-build/signed
+            --referrers-dest="$BASHBREW_REFERRERS_REPO"
```

Comment:
I notice the other invocation has both `--referrers-source` _and_ `--referrers-dest` -- does this one only having the latter mean that if we happened to build a fully reproducible image/object that we'd happily clobber the existing referrers for that image?  What if something besides this `image-signer-verifier` tool ends up creating referrers objects too?

Reply:
The reason why `verify` has `--referrers-source` is to specify where to pull attestations from if the referrers repo is not the same as the target repo. Similar for `--referrers-dest` for `sign`, they are both optional args to allow for separate referrers repo. 

Because `verify` is overloaded with signing operations you see both `source`/`dest` in the same command.

> does this one only having the latter mean that if we happened to build a fully reproducible image/object that we'd happily clobber the existing referrers for that image?

No, it will append the new attestations to the referrers manifest `source` is only used for verification. It does however overwrite the `:sha256-<digest>.att` tag. That tag is being used as a conduit to push the referrers artifacts. Currently, `:sha256-<digest>.att` only ever has one attestation and whichever one was pushed last has the tag but they all get returned by the referrers API.

> What if something besides this image-signer-verifier tool ends up creating referrers objects too?

They _should_ also append to the referrers manifest if configured to use the fallback method.